site/_posts/2017-01-23-last-week-in-manageiq-last-will-be-first.html.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Last Week in ManageIQ: The last will be the first!"
+author: hstastna
+date: 2017-01-23
+tags: LWIMIQ
+comments: true
+published: true
+---
+
+Hi all! [Hilda](https://github.com/hstastna) here. This is my first Last Week in ManageIQ post, and I hope this isn't my last! Below is a review of the last week's changes.
+
+## Featured
+
+As we are getting used to splitting repos, deleting so many lines and preparing for Angular 2 in the UI, there were 97 [PRs] merged last week in manageiq core, more than the week before, containing 226 [commits] by 43 [contributors]. You can choose from our list of [open issues], if you would like to contribute. Any help is appreciated.
```

Comment:
btw, github has some more problems regarding sorting and searching stuff there...